core/vm/absint_cfg_proof_check.go

```diff
@@ -87,7 +87,7 @@ func resolveCheck(sem *CfgAbsSem, code []byte, st0 *astate, pc0 int) (map[int]bo
 	succs := make(map[int]bool)
 	jumps := make(map[int]bool)
 
-	if opsem == nil || opsem.halts || opsem.reverts {
```

Comment:
This removal has to be verified

Reply:
LGTM. 

- `CfgOpSem` drops these fields, and there are no remaining usages of them. 
- The condition `opsem == nil` is a valid  and sufficient condition for returning because `toProgram` defines `stmt.ends = stmt.operation == nil`; so the expectation is consistent. https://github.com/ETCCooperative/erigon/pull/4/files#diff-632b254546eb0c76269d92705508d825a006ac5446f5d59dca3cfc13791c64b2R90